src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -1397,54 +1443,6 @@ public void updateCounts( final MoleculeCounts moleculeCounts,
             }
         }
 
-        protected ReadReport( final List<Interval> refCoverage, final List<SNV> snvList ) {
-            this.refCoverage = refCoverage;
-            this.snvList = snvList;
-        }
-
-        public static ReadReport NULL_REPORT = new ReadReport(new ArrayList<>(), new ArrayList<>());
-    }
-
-    @VisibleForTesting final static class PairedReadReport extends ReadReport {
-        final ReadReport report1;
-        final ReadReport report2;
-
-        public PairedReadReport( final ReadReport report1, final ReadReport report2 ) {
-            super(combineCoverage(report1, report2), combineVariations(report1, report2));
-            this.report1 = report1;
-            this.report2 = report2;
-        }
-
-        @Override
-        public boolean hasCleanFlanks( final int minFlankingLength, final int refLength ) {
```

Comment:
I don't find where this is reflected in the new merged single class.
Can you explain?

Reply:
I see, thanks!